docs/listeners.rst

```diff
@@ -75,21 +67,24 @@ every time a ``Crud.beforeRender`` event is emitted.
 The Callback
 ------------
 
-This method gets executed every time a ``Crud.beforeRender`` event is emitted
-from within CRUD or by you as a developer.
-
-When the event is emitted, we append a header to the client HTTP response named
-``X-Powered-By`` with the value ``CRUD 4.0``.
+This method gets executed every time a ``Crud.beforeRender`` event is emitted from within Crud or by you as a
+developer. When the event is emitted, we append a header to the client HTTP response named ``X-Powered-By`` with
+the value ``CRUD 4.0``.
 
 .. literalinclude:: _code/listener_example.php
 	 :language: php
 	 :linenos:
-	 :emphasize-lines: 18-26
+	 :emphasize-lines: 20-30
+
+Included listeners
+==================
 
-More on listeners
-^^^^^^^^^^^^^^^^^
+Crud comes with a selection of listeners included which cover some of the most common use-cases. These allow you to tap
```

Comment:
Redundant comes with/included, maybe something like:

Crud comes with a selection of listeners ready to cover most common use-cases.